.github/workflows/cla.yml

```diff
@@ -0,0 +1,49 @@
+#
+# A GitHub Action to manage Contributor License Agreement (CLA) compliance.
+# This action uses CLA Assistant (https://github.com/marketplace/actions/cla-assistant-lite) to ensure that
+# all contributors have signed the Deephaven CLA before their PRs are merged.  
+# The CLA can be signed by commenting on the PR.
+#
+# The CLA and all signatures are maintained at:  https://github.com/deephaven/cla
+# The "main" branch of this repository must not be protected.
+#
+# To use this action:
+# 
+# 1) Copy this file to `<your_project>/.github/workflows/cla.yml`
+#
+# 2) PERSONAL_ACCESS_TOKEN
+#
+# A personal access token must be set up to allow this bot to commit to the CLA repository.  To do this:
+# a) Create a personal access token with repo scope
+# b) Manually add the token to your repository's secrets as PERSONAL_ACCESS_TOKEN
+#
+#
+# 3) allowlist
+#
+# If a GitHub username is included in the allowlist, they will not be required to sign the CLA.  This is useful for allowing bots to contribute without a CLA.
+#
```

Comment:
I don't think we'll want to have this comment in our public code - it's mostly documentation of how we are supposed to use it...